chrisreams91/liftoff-group-project
# User profile

Created a User Profile Page:

-Added a first name input for users when they create an account

-Added a Profile link in the Nav Bar

-Created a Profile Page Controller including:
       -Home page with: name, username, profile picture, and task list
      -option to change their profile picture after they make their account (by using an input form @ picture.html)

-Created profile and picture thymeleaf templates


Testing: 
1. Create new account with first name
2. Go to user profile page
3. Make sure the page reflects the name & username of the account you created
4. Change your profile picture. It should save to MySQL
